public/static/locales/en/events.json

```diff
@@ -27,7 +27,7 @@
     "labelPlaces": "Places",
     "labelCategories": "Categories",
     "labelTargetGroups": "Target groups",
-    "labelOtherClassification": "Other classification",
+    "labelOtherClassification": "Activities",
```

Comment:
`labelActivities` would be more convenient key name?

Reply:
absolutely, fixed :)